server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -950,11 +952,10 @@ private SortedMap<TServerInstance,TabletServerStatus> gatherTableInformation(
       Set<TServerInstance> currentServers, SortedMap<TabletServerId,TServerStatus> balancerMap) {
     final long rpcTimeout = getConfiguration().getTimeInMillis(Property.GENERAL_RPC_TIMEOUT);
     int threads = getConfiguration().getCount(Property.MANAGER_STATUS_THREAD_POOL_SIZE);
-    ExecutorService tp = ThreadPools.getServerThreadPools()
```

Comment:
Its unrelated to this PR, but I think this used to be a thread pool of unlimited size.  I need to look back and see what changed.  There is a comment a bit further down in the code about rate limiting because its an unbounded thread pool.

Reply:
It looks like this is handled in ThreadPools. When the value of Property.MANAGER_STATUS_THREAD_POOL_SIZE is zero it returns an unbounded thread pool.